server/macros/__init__.py

```diff
@@ -4,5 +4,22 @@
 
 Use `superdesk.macros.macros.register` for registration.
 """
+import sys
+import imp
+macros = ['macros.currency', 'macros.abstract_populator']
 
-from . import currency  # noqa
+try:
```

Comment:
I'm not sure this part is required. If you reload this module it will reload also modules within it no?